<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Imagem</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="input_arquivo"/>
          </item>
          <item>
           <widget class="QPushButton" name="botao_buscarImagem">
            <property name="text">
             <string>Buscar Imagem</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="check_mostrarOriginal">
            <property name="text">
             <string>Mostrar original</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_imagem">
          <property name="text">
           <string>A imagem vai aqui. Setar como vazio</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Filtros espaciais</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_filtrosEspaciais">
            <item>
             <property name="text">
              <string>Nenhum</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Filtros no domínio da frequência</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_filtrosFrequencia">
            <item>
             <property name="text">
              <string>Nenhum</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="botao_aplicar">
            <property name="text">
             <string>Aplicar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>563</width>
             <height>513</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Transformações geométricas</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_escala">
              <property name="text">
               <string>Escala</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="valor_escala">
              <property name="text">
               <string>1.0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slide_escala">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_angulo">
              <property name="text">
               <string>Ângulo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="valor_angulo">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="slide_angulo">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_espelhamento">
              <property name="text">
               <string>Espelhamento</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_espelhamentoH">
              <property name="text">
               <string>Horizontal</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_espelhamentoV">
              <property name="text">
               <string>Vertical</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_espelhamentoHV">
              <property name="text">
               <string>Horizontal e vertical</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="check_translacao">
              <property name="text">
               <string>Translação</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_translacaoCE">
              <property name="text">
               <string>Cima e esquerda</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_translacaoBD">
              <property name="text">
               <string>Baixo e direita</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
